### ISSUE TITLE
Image is not showing from few days

### ISSUE THREAD

**Issue by venuscreats** [barryvdh/laravel-dompdf#695]
hello all,
I using this amazing script and it worked well and found some image missing in view. when I check that it was ok with my localhost and issue with live server. Image paths ok and tried below ways.

base_url
public url 
assest path
absolute path 

but image from live server will not show. But when I tried direct link to image in another server it will work well.  tried both png and jpg formats. all are ok with another server. How to fix this? 

Thanks,

**Comment by sayhicoelho** [comment]
Here when trying to show <img src="{{ asset(...) }} .../> the server fires an "max_execution_time" exception. How to show images in the body?

**Comment by sayhicoelho** [comment]
Fixed here https://github.com/barryvdh/laravel-dompdf/issues/303#issuecomment-470950309

**Comment by venuscreats** [comment]
> Here when trying to show `<img src="{{ asset(...) }} .../>` the server fires an **max_execution_time** exception. How to show images in the body?

Thanks for reply. Yes I tried many ways. early used this way and worked well.

<img src="{{ url('/') }}/public/upload/avatar/162_1573098815_200x200.jpg" width="80" height="80" style="display: block; margin-left: 0px; width: 60px; height: 100px; " />

also tried public_path, asset, too. but with many testings I found these.

1. there are another few images in this PDF and able to duplicate using current path. it is working.
2. Images from another server and direct path are working fine.

Is there any update or something to do?
